authorizer.py

```diff
@@ -85,6 +87,9 @@ def _VerifyServingSession(self, institution, session):
     if (serving_session.login_type == "schedule" and
         self.handler.request.path == "/spots_available"):
```

Comment:
Is there any way to invert this so that redirect always happens unless checks for access and session are met. I can see how someone (very possibly me in the future) might forget or not realize they need to do this check on every page.